2025-05-12 10:47:07 [error]: Prisma Error: 
Invalid `prisma.user.findUnique()` invocation:


Error validating datasource `db`: the URL must start with the protocol `prisma://`
2025-05-12 10:51:18 [error]: Prisma Error: 
Invalid `prisma.user.findUnique()` invocation:


Error validating datasource `db`: the URL must start with the protocol `prisma://`
2025-05-12 12:37:05 [error]: Prisma Error: Violation of UNIQUE KEY constraint 'User_oauthProvider_oauthId_key'. Cannot insert duplicate key in object 'dbo.User'. The duplicate key value is (<NULL>, <NULL>).
2025-05-12 12:37:05 [error]: Prisma Error: 
Invalid `prisma.user.create()` invocation in
C:\Projects\syf\server-prisma\src\services\auth.service.ts:15:38

  12 try {
  13   // Add try...catch inside createUser
  14   const hashedPassword = await bcrypt.hash(password, 12)
→ 15   const user = await prisma.user.create(
Unique constraint failed on the constraint: `dbo.User`
2025-05-12 12:37:31 [error]: Prisma Error: Violation of UNIQUE KEY constraint 'User_oauthProvider_oauthId_key'. Cannot insert duplicate key in object 'dbo.User'. The duplicate key value is (<NULL>, <NULL>).
2025-05-12 12:37:31 [error]: Prisma Error: 
Invalid `prisma.user.create()` invocation in
C:\Projects\syf\server-prisma\src\services\auth.service.ts:15:38

  12 try {
  13   // Add try...catch inside createUser
  14   const hashedPassword = await bcrypt.hash(password, 12)
→ 15   const user = await prisma.user.create(
Unique constraint failed on the constraint: `dbo.User`
2025-05-12 12:37:33 [error]: Prisma Error: Violation of UNIQUE KEY constraint 'User_oauthProvider_oauthId_key'. Cannot insert duplicate key in object 'dbo.User'. The duplicate key value is (<NULL>, <NULL>).
2025-05-12 12:37:33 [error]: Prisma Error: 
Invalid `prisma.user.create()` invocation in
C:\Projects\syf\server-prisma\src\services\auth.service.ts:15:38

  12 try {
  13   // Add try...catch inside createUser
  14   const hashedPassword = await bcrypt.hash(password, 12)
→ 15   const user = await prisma.user.create(
Unique constraint failed on the constraint: `dbo.User`
2025-05-12 12:38:30 [error]: Prisma Error: Violation of UNIQUE KEY constraint 'User_oauthProvider_oauthId_key'. Cannot insert duplicate key in object 'dbo.User'. The duplicate key value is (<NULL>, <NULL>).
2025-05-12 12:38:30 [error]: Prisma Error: 
Invalid `prisma.user.create()` invocation in
C:\Projects\syf\server-prisma\src\services\auth.service.ts:15:38

  12 try {
  13   // Add try...catch inside createUser
  14   const hashedPassword = await bcrypt.hash(password, 12)
→ 15   const user = await prisma.user.create(
Unique constraint failed on the constraint: `dbo.User`
2025-05-12 12:38:59 [error]: Prisma Error: Violation of UNIQUE KEY constraint 'User_oauthProvider_oauthId_key'. Cannot insert duplicate key in object 'dbo.User'. The duplicate key value is (<NULL>, <NULL>).
2025-05-12 12:38:59 [error]: Prisma Error: 
Invalid `prisma.user.create()` invocation in
C:\Projects\syf\server-prisma\src\services\auth.service.ts:15:38

  12 try {
  13   // Add try...catch inside createUser
  14   const hashedPassword = await bcrypt.hash(password, 12)
→ 15   const user = await prisma.user.create(
Unique constraint failed on the constraint: `dbo.User`
2025-05-12 12:42:43 [error]: Prisma Error: Violation of UNIQUE KEY constraint 'User_oauthProvider_oauthId_key'. Cannot insert duplicate key in object 'dbo.User'. The duplicate key value is (<NULL>, <NULL>).
2025-05-12 12:42:43 [error]: Prisma Error: 
Invalid `prisma.user.create()` invocation in
C:\Projects\syf\server-prisma\src\services\auth.service.ts:15:38

  12 try {
  13   // Add try...catch inside createUser
  14   const hashedPassword = await bcrypt.hash(password, 12)
→ 15   const user = await prisma.user.create(
Unique constraint failed on the constraint: `dbo.User`
2025-05-12 15:19:43 [error]: Prisma Error: 
Invalid `prisma.user.findUnique()` invocation in
C:\Projects\syf\server-prisma\src\controllers\auth.controller.ts:14:46

  11 static async register(req: Request, res: Response): Promise<void> {
  12   try {
  13     const { email, password } = res.locals.sanitized.body
→ 14     const existingUser = await prisma.user.findUnique(
Can't reach database server at `localhost:1433`

Please make sure your database server is running at `localhost:1433`.
2025-05-12 16:41:32 [error]: Prisma Error: Invalid object name 'dbo.User'.
2025-05-12 16:41:32 [error]: Prisma Error: 
Invalid `prisma.user.findUnique()` invocation in
C:\Projects\syf\server-prisma\src\controllers\auth.controller.ts:14:46

  11 static async register(req: Request, res: Response): Promise<void> {
  12   try {
  13     const { email, password } = res.locals.sanitized.body
→ 14     const existingUser = await prisma.user.findUnique(
The table `dbo.User` does not exist in the current database.
